react-multiple-components/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import './RotatingBanner.css';
+import { Title } from './Title';
+import { Next } from './Next';
+import { ButtonRow } from './ButtonRow';
+import { Prev } from './Prev';
+import { useState } from 'react';
+
+type Props = {
+  items: string[];
+};
+
+export function RotatingBanner({ items }: Props) {
+  const [stage, setStage] = useState(0);
+  function handleNext() {
+    setStage((stage + 1) % 6);
+  }
+  function handlePrev() {
+    setStage((6 + stage - 1) % 6); // I add 6 here so that stage doesn't change from 0 to -1, but rather 0 to 5 % 6, which is 5.
```

Comment:
This is good, but I suggest you use items.length here (and above) so it still works even if there are not exactly 6 items.

Reply:
Wow, I didn't think I'd make this mistake again. I went back and modified it this time. I hope I'll build a good habit.